sktime/forecasting/boxcox_bias_adjusted_forecaster.py

```diff
@@ -0,0 +1,188 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+"""BoxCoxBiasAdjustedForecaster implementation.
+
+This module implements a forecaster that applies Box-Cox transformation
```

Comment:
this docstring is not visible in the API reference - could you kindly move it to the docstring of the class, from where it will be displayed directly in the docs?

Reply:
Sure. Working on it.